app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -12,14 +12,19 @@ import android.view.accessibility.AccessibilityManager
 import androidx.preference.PreferenceManager
 import mozilla.components.concept.engine.Engine
 import mozilla.components.concept.engine.EngineSession
+import mozilla.components.feature.sitepermissions.SitePermissionsRules
 import org.mozilla.focus.R
+import org.mozilla.focus.ext.components
 import org.mozilla.focus.fragment.FirstrunFragment
 import org.mozilla.focus.searchsuggestions.SearchSuggestionsPreferences
 
 /**
  * A simple wrapper for SharedPreferences that makes reading preference a little bit easier.
  */
-@Suppress("TooManyFunctions", "LargeClass") // This class is designed to have a lot of (simple) functions
+@Suppress(
+    "TooManyFunctions",
+    "LargeClass"
+) // This class is designed to have a lot of (simple) functions
```

Comment:
Nit: I think if we would like to add extra documentation, we should add it above the class. With the other docs :) 
```
/**
 * A simple wrapper for SharedPreferences that makes reading preference a little bit easier.
 */
```